assimilation_code/programs/run_tests.csh

```diff
@@ -186,9 +186,9 @@ foreach PROGRAMDIRECTORY ( $DO_THESE_DIRECTORIES )
 
 #      echo
 #      echo
-#      echo "=================================================================="
-#      echo "Running tests for $PROGRAMDIRECTORY"
-#      echo "=================================================================="
+#      echo "------------------------------------------------------------------"
+#      echo "Running tests for $PROGRAMDIRECTORY starting at "`date`
+#      echo "------------------------------------------------------------------"
```

Comment:
The equal-sign delimeter is used by test_dart.csh to demark the start/stop of each run_test.csh script.
Within the run_test scripts, I am trying to use the dash-delimiter.  Maybe we don't need anything given the amount of whitespace. Whaddya think?

Reply:
what's the eventual goal?  if it's for tests to be run automatically and the results parsed into a web page or some summary format, then remove all the delimiters and whitespace and add unique strings for each section/test/error msg.  that makes it easier for a program to parse the output.  

the delimiters are useful for people if they're reading the output, but if the length of the test log is too much for a person to scan then they don't accomplish much.  

in my opinion it's not worth changing right now -- there may be better ways to run automated tests.  but if these scripts are going to be used, then when someone overhauls the scripts they should look at the format of the output being generated and write a post-processing parser for it at the same time.